packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -8,6 +8,11 @@ export interface ISearchBox {
    * Sets focus inside the search input box.
    */
   focus(): void;
+
+  /**
+   * Returns a flag indicating whether the search box has focus
```

Comment:
small nit, but could simply say "Returns whether or not the SearchBox has focus".